server/conf/logback.xml

```diff
@@ -31,7 +31,8 @@
   <logger name="s3client" level="INFO" />
   <logger name="io.ebean" level="INFO" />
   <logger name="org.pac4j" level="INFO" />
-  <logger name="auth" level="INFO" />
+  <logger name="auth" level="DEBUG" />
+  <logger name="org.pac4j" level="DEBUG" />
```

Comment:
suggestion (blocking): this will probably be too much auth-related logs to be a good default in dev

Reply:
removed